src/main/java/org/sqlite/SQLiteConfig.java

```diff
@@ -862,11 +862,11 @@ public void useLegacyFileFormat(boolean use) {
      * earlier.When the flag is off, using the ALTER TABLE RENAME command will mean that all
      * references to the table anywhere in the schema will be converted to the new name.
      *
-     * @param use True to turn on legacy alter table behaviour; false to turn off.
+     * @param flag True to turn on legacy alter table behaviour; false to turn off.
      * @see <a href="https://www.sqlite.org/pragma.html#pragma_legacy_alter_table</a>
      */
-    public void setLegacyAlterTableFlag(boolean use) {
-        set(Pragma.LEGACY_ALTER_TABLE, use);
+    public void setLegacyAlterTableFlag(boolean flag) {
```

Comment:
can you also change the name from `setLegacyAlterTableFlag` to `setLegacyAlterTable` to be consistent with other methods?

Reply:
my bad didn't spot the method rename